marin/evaluation/evaluators/lm_evaluation_harness_evaluator.py

```diff
@@ -83,6 +83,7 @@ def evaluate(
                     tasks=[eval_task.name],
                     num_fewshot=eval_task.num_fewshot,
                     model_args=pretrained_args,
+                    apply_chat_template=True,
```

Comment:
wait, should this always be set? i don't think so right?

Reply:
Want to make sure we're not breaking things...like this should not be set for base model evals, right?  Have to be careful here